pipelines/release.yml

```diff
@@ -1228,8 +1228,8 @@ resources:
   type: git
   icon: *git-icon
   source:
-    uri: https://github.com/concourse/concourse-chart.git
-    branch: release/((release_minor)).x
+    uri: https://github.com/helm/charts
+    branch: master
```

Comment:
@taylorsilva heey, wdyt of this approach? (context here: https://github.com/concourse/ci/issues/222#issuecomment-559174540)

Reply:
This approach makes sense 👍